move-vm-backend/src/types.rs

```diff
@@ -72,6 +73,47 @@ impl VmResult {
     }
 }
 
+/// Inner MoveVM gas handling multiplier.
```

Comment:
It may be worth to write two sentences why we are using this multiplier.
There are many places below when we're using it but there is no explanation.

Reply:
Done